releases.md

```diff
@@ -33,10 +33,11 @@ As of 2018-11-16, PELUX 3.0 has been released! Documentation can be found
   is therefore not functioning as intended.
 * When running the QtAS versions of the platform, there are no ttys allocated on
   the terminal. To get a terminal, ssh to the target. If you don't know the IP
-  address of your target, use arp-scan to search for it on your network.
-* There is going to be minor hickups in the PELUX Software Factory Baseline that
-  we have missed even though we've proofread them. Please point them out and we
-  will fix them!
+  address of your target, use arp-scan to search for it on your network. You can
+  also select the Qt button at the bottom right corner of the home screen and
+  scroll down to see the IP address.
+* Qt Auto Neptune image on Rasperry Pi 3 is broken: All apps are not accessible
```

Comment:
```suggestion
 * Qt Auto Neptune image on Rasperry Pi 3 contains known bugs: All apps are not accessible
```

Reply:
propose change to "is not working properly"